index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+  <link rel="stylesheet" type="text/css" href="css/style.css" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous" />
+  <title>Trial review</title>
+</head>
+
+<body>
+  <ul class="nav justify-content-end">
+    <li class="nav-item">
+      <a class="nav-link active" aria-current="page" href="#">Home</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="#">Work</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="#">Contact</a>
+    </li>
+  </ul>
+
+  <section class="section-1">
+    <div class="content1">
+      <h1 id="h1-1">Hey We are Donard & Zaliha</h1>
+      <h4 id="h4-1">Software Developers</h4>
+    </div>
+  </section>
+
+  <section class="section-2">
+    <h3>There are some of my projects</h3>
+
+    <div class="row row-cols-1 row-cols-md-3 g-4">
+      <div class="col">
+        <div class="card">
+          <img src="images/normanborlaug.jpeg" class="card-img-top" alt="..." />
+          <div class="card-body">
+            <h6 class="card-title">Tribute Page</h6>
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card">
+          <img src="images/random-quote-machine.png" class="card-img-top" alt="..." />
+          <div class="card-body">
+            <h6 class="card-title">Random Quote Machine</h6>
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card">
+          <img src="images/calculator.png" class="card-img-top" alt="..." />
+          <div class="card-body">
+            <h6 class="card-title">Javascript Calculator</h6>
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card">
+          <img src="images/mapacross.png" class="card-img-top" alt="..." />
+          <div class="card-body">
+            <h6 class="card-titler">Map Data Accross The Globe</h6>
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <div class="card">
+          <img src="images/wikipedia.png" class="card-img-top" alt="..." />
+          <div class="card-body">

```

Comment:
- [ ]  Kindly add text to all alt tags this helps your website to be more accessible